streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -61,7 +61,7 @@ public StandbyTask(TaskId id,
         // initialize the topology with its own context
         this.processorContext = new StandbyContextImpl(id, applicationId, config, stateMgr, metrics);
 
-        initializeStateStores();
+        registerAndinitializeStateStores();
```

Comment:
I understand standby tasks have different initialization process, but not sure why this enforce us to make registration and initialization together?